vote_text_drafts/clarify_RPIP-4.md

```diff
@@ -0,0 +1,42 @@
+> Vote options:
```

Comment:
Not to make even more complex... But should there be a third option?

"Nodes with effectively staked RPL as the protocol defines it at any given time shall be eligible voters"

Both current options freeze the determination regardless of if the effectively staked requirements change in the future.

Reply:
Alternatively, a: "Something else" option.

Reply:
Some discussion on discord https://discord.com/channels/405159462932971535/1157351380999225465/1158774925604163634

I don't believe this would be beneficial. I'm happy to be convinced if there's community leaning.